packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -326,7 +326,7 @@ export default defineComponent({
     },
     viewMode: {
       type: String,
-      default: ViewModeConstants.default
+      default: ViewModeConstants.default.name
```

Comment:
You have already introduced the `defaultModeName` const for this

Reply:
Apparently slipped through, sry about that